core/templates/pages/exploration-editor-page/services/exploration-tags.service.spec.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit test for Exploration Tags Service
+ */
+
+require('pages/exploration-editor-page/services/exploration-tags.service.ts');
+
+describe ('Exploration Tag Service', () =>{
+  beforeEach(angular.mock.module('oppia'));
+  var component=null;
```

Comment:
Move it to line 22 (before the first beforeEach) and changes it name.
```suggestion
  var ExplorationTagsService = null;
```